contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -31,7 +31,7 @@ script: |
   set -e -o pipefail
 
   WRAP_DIR=$HOME/wrapped
-  HOSTS="i686-w64-mingw32 x86_64-w64-mingw32"
+  HOSTS="x86_64-w64-mingw32"
```

Comment:
This looks so bare. Almost makes me want to build a PPC64 Windows binary just for the sake of it. :)

Reply:
Can you install windows on ppc even?

Reply:
NT4 :p

Reply:
o dear I have been running bitcoin core on a 32-bit windows system for a few years now. Am I the last one? I'd like to upgrade to 0.18 please. Any chance of having 32-bit windows back, or is it too much bother?

Reply:
Why can't you use the 64-bit build?

Reply:
I'll defer to someone who knows more than me, but can a 64-bit build run on a 32-bit machine?